### PR TITLE
Moltres slides

### PR DESCRIPTION
**Pull request by smpark7**
Addresses #12

### CODE REVIEW

File: moltres.tex [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+			neutronics/thermal-hydraulics simulations of \glspl{MSR}
+			\begin{itemize}
+				\item Multi-group neutron diffusion (arbitrary groups)
+				\item Delayed neutron precursor advection
+				\item Incompressible Navier-Stokes for temperature
+				advection-diffusion
+				\item 2D axisymmetric or 3D modeling
+			\end{itemize}
+		\end{itemize}
+%		\begin{figure}
```
Comment (samgdotson) [inline moltres.tex]: Delete the commented lines if they're used in the very next slide.

**Review by samgdotson — APPROVED** [review]
Straggling commented lines, but otherwise :+1:

File: moltres.tex (hunk 1 above)
Comment (smpark7) [reply moltres.tex]: oh thanks I forgot about those

**Comment by samgdotson** [comment]
Looks good

**Pull request merged**